### PR TITLE
feat: add int64 support for arff files

### PR DESCRIPTION
**Pull request by mllg**

### CODE REVIEW

File: R/read_arff.R [hunk 1]
```diff
@@ -120,7 +120,15 @@ read_arff = function(path) {
 
   for (j in which(col_classes == "integer")) {
     x = data[[j]]
-    if (!is.integer(x) && test_integerish(x)) {
+
+    if (inherits(x, "integer64")) {
+      print(j)
```
Comment (sebffischer) [inline R/read_arff.R:125]: remove print statement

**Comment by mllg** [comment]
I've just noticed that bit64 is required - but I'm unsure if this has to go into suggests or imports ... looks like data.table reads in the data just fine with `fread()`, but then failing in `between()`?

**Review by sebffischer — APPROVED** [review]

**Comment by sebffischer** [comment]
If the int64 case is rare then I would put it into Suggests. What happened to these columns berfore this fix?

**Comment by sebffischer** [comment]
Also did you check what happens with the parquet files?

**Pull request merged**